### PR TITLE
Fix broken code in docs

### PR DESCRIPTION
**Pull request by TezRomacH**
## What does this PR do?

Fixes code in docs

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [ ] Did you write any new necessary tests? (not for typos and docs)
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [ ] Make sure the title is self-explanatory and the description concisely explains the PR
 - [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [ ] **Check that target branch and milestone match!**

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by Borda — APPROVED** [review]

File: docs/source/datamodules.rst [hunk 1]
```diff
@@ -75,8 +75,9 @@ The equivalent DataModule just organizes the same exact code, but makes it reusa
 
     class MNISTDataModule(pl.LightningDataModule):
 
-        def __init__(self, data_dir: str = PATH, batch_size):
+        def __init__(self, data_dir: str = PATH, batch_size: int = BATCH_SIZE):
```
Comment (carmocca) [inline docs/source/datamodules.rst:78]: batch_size should be lowercase

**Comment by Borda** [comment]
@TezRomacH there won't be any more releases in 1.1.x so I have changed your destination branch to feat 1.2
pls, rebase on the target branch, or maybe easier just cherry-pick the commit (@kaushikb11 will help you if needed)

File: docs/source/datamodules.rst (hunk 1 above)
Comment (TezRomacH) [reply docs/source/datamodules.rst]: Done :)

**Comment by TezRomacH** [comment]
> @TezRomacH there won't be any more releases in 1.1.x so I have changed your destination branch to feat 1.2
> pls, there is no need to close this PR, just rebase on the target branch, or maybe easier just cherry-pick the commit (@kaushikb11 will help you if needed)

I did a rebase, now it should work.

**Review by SkafteNicki — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

**Pull request merged**